mlflow/onnx.py

```diff
@@ -157,7 +157,7 @@ def save_model(
     model_data_path = os.path.join(path, model_data_subpath)
 
     # Save onnx-model
-    onnx.save_model(onnx_model, model_data_path)
+    onnx.save_model(onnx_model, model_data_path, save_as_external_data=True)
```

Comment:
Can we add a test case to verify that we're still able to use `mlflow.onnx.load_model()` to load models that were not saved with `save_as_external_data=True`?